app/javascript/shared/helpers/joinUrl.js

```diff
@@ -0,0 +1,32 @@
+const isValidPath = path => {
+  if (!path) return false;
+  if (path === null && path === '') return false;
+  // if path is just a sequence of slashes
+  if (/^\/+$/.test(path)) return false;
+
+  return true;
+};
+
+/**
+ * Join multiple paths together with a base URL
+ * NOTE: This function is not designed to handle query strings or fragments
+ *
+ * @param {string} baseUrl - The base URL to join the paths to
+ * @param  {...string} paths - The paths to join
+ * @returns {string} - The full URL
+ */
+export function joinUrl(baseUrl, ...paths) {
```

Comment:
@scmmishra Are we using this method anywhere?

Reply:
Nah, was used earlier, removed it